### PR TITLE
add pcre

### LINKED ISSUE #63: Add pcre to git build
The `git` recipe currently does not include `pcre` as a dependency.  This results in 

```
git grep -P <whatever>

fatal: cannot use Perl-compatible regexes when not compiled with USE_LIBPCRE
```

We should add `pcre` as a dependence so that the `-P` option on commands will work.

### PR DESCRIPTION
**Pull request by timsnyder**
add pcre dependency so that the '-P' option of git-log and
git-grep will work

fixes #63

Checklist
* [X] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by timsnyder** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -39,6 +40,7 @@ requirements:
     - gettext   # [unix]
     - libiconv  # [unix]
     - openssl   # [unix]
+    - pcre      # []unix
```
Comment (scopatz) [inline recipe/meta.yaml]: This is a typo on the selector.  Can you please send in a new PR with the fix.

Comment (timsnyder) [reply recipe/meta.yaml]: Yes.  I just noticed this myself.  Would it be horrible to fix it in #65 ?  

I don't mind making it a separate PR.

Comment (timsnyder) [reply recipe/meta.yaml]: @scopatz, I pushed a fix to #65 as https://github.com/conda-forge/git-feedstock/pull/65/commits/5c5c21379e9cdb67686ab3918552af0ebecbc2e6.

If you'd rather I break it out into a separate PR, I'll revert it from #65 .

Comment (scopatz) [reply recipe/meta.yaml]: Ok thanks, that's fine

**Pull request merged**